README.adoc

```diff
@@ -320,7 +320,9 @@ The actuator exposes the following:
 
 NOTE: There is also an `/actuator/shutdown` endpoint, but, by default, it is visible only
 through JMX. To http://docs.spring.io/spring-boot/docs/{spring_boot_version}/reference/htmlsingle/#production-ready-endpoints-enabling-endpoints[enable it as an HTTP endpoint], add
-`management.endpoints.web.exposure.include=health,info,shutdown` to your `application.properties` file.
+`management.endpoint.shutdown.enabled=true`
```

Comment:
Add "to your `application.properties` file" to this line. (Do not add the period.)

Reply:
Agreed